### PR TITLE
Add Alpaca Broker

### PR DESCRIPTION
**Pull request by jmaslek**
This PR adds alpaca broker with the same hold and historical portfolio functions.  

Note I haven't invested in this account enough to show the alphist.  I just bought some penny stocks to test out, so I can update the readme in like a week with all the loss porn.

### CODE REVIEW

**Comment by DidierRLopes** [comment]
So, this may be a bit futuristic. But the way I see this is the following:
* Having only 2 commands: hold, hist

PS: You can change hist to historical, as you suggested the other day. It made sense to me.

When you do login to rh, you get:
Current brokers: rh

This means that if you do hold or hist, it's pointing at "robinhood".

BUT, imagine that you have:
Current brokers: rh alp

Then, if you did hold or hist, it would be pointing at both and you would show the combined holdings or historical.

**Comment by jmaslek** [comment]
So I was thinking something along that line as well.  I was going to play with merging them later this week (these are the only two that I actually have).  I think looking individually is good as well (I can bury the menu unless specified).

For the future, my thought process was going to be something like 

- `login brokers`
- `merge`
- `hold` or `hist`

So the command to merge all would be stated specifically and then allow for the options of hold/hist (still allowing individual hold/hist).

For the future as well, I am noticing that the api's return different types of data (alpaca has a nice historical portfolio get, but robin-stocks just returns high low of a period, not equity at a given time), so that is why I submitted a PR that just had this broker.

File: gamestonk_terminal/portfolio/README.md [hunk 1]
```diff
@@ -60,4 +70,39 @@ usage: rhhist [-s --span] [-i --interval]
     
 Example Default Output:
 
-![rhhist](https://user-images.githubusercontent.com/18151143/111718919-36da8e00-8831-11eb-99e1-957c8eccb583.png)
\ No newline at end of file
+![rhhist](https://user-images.githubusercontent.com/18151143/111718919-36da8e00-8831-11eb-99e1-957c8eccb583.png)
+## ALPACA <a name="ALPACA"></a>
+
+Alpaca has a nicely maintained python API that is used here.  To login, the best approach is to just save the following environment variables:
+* "APCA_API_BASE_URL"
```
Comment (DidierRLopes) [inline gamestonk_terminal/portfolio/README.md:77]: To follow others API Keys, probably use something like: GT_ALP_BASE/KEY/SECRET

**Review by DidierRLopes — APPROVED** [review]
Agreed with what you mentioned, we're on the right path here!

Just a minor comment. All the rest looks good to me! Feel free to merge it

File: gamestonk_terminal/portfolio/README.md (hunk 1 above)
Comment (jmaslek) [reply gamestonk_terminal/portfolio/README.md:77]: So I was going to keep it like that, but the alpaca api actually goes in and looks for those specific env variables.  Anyone using Alpaca probably has those stored as well.  

I can change it to keep the same convention if you would like.

Also - yes reqs are updated (from yesterday)

Comment (DidierRLopes) [reply gamestonk_terminal/portfolio/README.md:77]: On that basis, I agree with you to leave it as is! Feel free to merge it

**Pull request merged**